### ISSUE TITLE
'make all' fails and jenkins with it (go-bindata)

### ISSUE THREAD

**Issue by liamsi** [tendermint/go-crypto#95]
Jenkins runs `make all` which fails (locally too).
```
make all                                                                                                                                         
Makefile:31: *** "No go-bindata in PATH".  Stop.
```

**Comment by liamsi** [comment]
target `all` should include `get_tools`

Be aware that the repo owner of go-bindata changed:

https://www.reddit.com/r/golang/comments/7vv9zz/popular_lib_gobindata_removed_from_github_or_why/
and
https://github.com/jteeuwen/go-bindata/issues/5

**Comment by ebuchman** [comment]
:o 

Let's not use that and instead just hardcode the output ...

**Comment by liamsi** [comment]
resolved by #101
